tests-integration/Cargo.toml

```diff
@@ -0,0 +1,62 @@
+[package]
+name = "tests-integration"
+version = "0.1.0"
+authors = { workspace = true }
+license = { workspace = true }
+edition = { workspace = true }
+repository = { workspace = true }
+readme = "README.md"
+keywords = [ "blockchain", "consensus", "cosmos", "ibc", "tendermint" ]
+publish = false
+description = """
+    Integration tests that make use of the types exposed by `ibc-testkit`. These tests also
+    depend upon the other IBC crates. They live in a separate crate that is not meant to be published
+    so as to avoid circular dependencies.
+"""
+
+[dependencies]
+# external dependencies
+doc-comment     = "0.3.3"
+serde           = { workspace = true, optional = true }
+serde_json      = { workspace = true, optional = true }
+subtle-encoding = { workspace = true }
+tracing         = { version = "0.1.40", default-features = false }
+
+# ibc dependencies
+ibc                         = { workspace = true, features = [ "std" ] }
+ibc-core-client-types       = { workspace = true }
+ibc-core-commitment-types   = { workspace = true }
+ibc-client-cw               = { workspace = true }
+ibc-client-tendermint-cw    = { workspace = true }
+ibc-client-tendermint-types = { workspace = true }
+ibc-core-host-types         = { workspace = true }
+ibc-primitives              = { workspace = true }
+ibc-query                   = { workspace = true }
+ibc-testkit                 = { workspace = true }
+
+# basecoin dependencies
+basecoin-store = { version = "0.1.0" }
+
+# cosmos dependencies
+tendermint         = { workspace = true }
+tendermint-testgen = { workspace = true }
+
+[dev_dependencies]
+cosmwasm-std   = { workspace = true }
+hex            = { version = "0.4.2" }
+rstest         = { workspace = true }
+test-log       = { version = "0.2.13", features = [ "trace" ] }
+tendermint-rpc = { workspace = true }
+
+[features]
+default = [ "std" ]
+std = [
+  "serde/std",
+  "serde_json/std",
+  "ibc/std",
+  "tendermint/std",
+  "dep:serde",
+  "ibc/serde",
+  "ibc-testkit/serde",
+  "serde_json",
```

Comment:
should it be `"dep:serde_json"`?

Reply:
I'm not sure. Should we add `dep:serde_json` while also keeping `dep:serde`?

Reply:
`serde` and `serde_json` are two different (optional) dependencies. I don't think, enabling `serde` dependency will enable `serde_json` dependency.

Ref: [cargo-book](https://doc.rust-lang.org/cargo/reference/features.html#optional-dependencies)

Reply:
[09bf968](https://github.com/cosmos/ibc-rs/pull/1233/commits/09bf96897193bddb77ea4024ce9a9f60e7a2b1c5)